.github/workflows/repo2cpe-download.yaml

```diff
@@ -23,14 +26,19 @@ jobs:
     - name: Download repository mappings
       run: |
         curl --fail --silent --show-error --max-time 60 --retry 3 --create-dirs \
-          -o 'redhat-repository-mappings/#1' \
+          -o 'repomapping-tmp/#1' \
           'https://access.redhat.com/security/data/metrics/{repository-to-cpe.json,container-name-repos-map.json}'
-        for f in redhat-repository-mappings/*; do
+        for f in repomapping-tmp/*; do
           jq empty "$f"
         done
 
     - name: Upload repository mappings to Google Cloud Storage
       run: |
+        # Zip the files into repomapping.zip
+        zip repomapping.zip repomapping-tmp/*
```

Comment:
Can you evaluate if we can get smaller files with gzip (or zstd, if it's installed)? Similarly, what is the size of this file? Just so we have an idea of the amount of data we will be transferring into a cluster

Reply:
GZIP is worse. Zstd is faster and achieves a higher compression ratio. Still, the gain is probably negligible for these files, so it is better to think about compatibility and reliability when creating/reading the data rather than performance. I feel it won't matter.

Reply:
Combine the two JSON files into a single ZIP file with a compressed size of 103.9 KB. So I think zip file should be good. @RTann

Reply:
oh this is the mapping files, not the vulnerability data. ZIP works just fine then